src/sources/soundsourceflac.cpp

```diff
@@ -100,7 +100,17 @@ SoundSource::OpenResult SoundSourceFLAC::tryOpen(
     }
     FLAC__stream_decoder_set_md5_checking(m_decoder, false);
     const FLAC__StreamDecoderInitStatus initStatus(
-            FLAC__stream_decoder_init_stream(m_decoder, FLAC_read_cb, FLAC_seek_cb, FLAC_tell_cb, FLAC_length_cb, FLAC_eof_cb, FLAC_write_cb, FLAC_metadata_cb, FLAC_error_cb, this));
+            FLAC__stream_decoder_init_stream(
+                    m_decoder,
+                    FLAC_read_cb,
+                    FLAC_seek_cb,
+                    FLAC_tell_cb,
+                    FLAC_length_cb,
+                    FLAC_eof_cb,
+                    FLAC_write_cb,
+                    FLAC_metadata_cb,
+                    FLAC_error_cb,
+                    this));
```

Comment:
Did you put all the line brakes manually? I think I like It. Originally I would have done it like this: 

```
LAC__stream_decoder_init_stream(m_decoder, FLAC_read_cb, FLAC_seek_cb, FLAC_tell_cb, FLAC_length_cb,
        FLAC_eof_cb, FLAC_write_cb, FLAC_metadata_cb, FLAC_error_cb, this));
```

Should we allow both? Are we able to define a guide, when do what?

Reply:
This is a custom decision each developer has to make. We didn't have rules for this in the past.

Reply:
Clang forces one or the other?
Than all is fine for me.

Reply:
Yes. It the decision is made depending on the context. If you place at least two arguments on the same line ClangFormat seems to assume that it may place all arguments on a single line.

The only exception is if all arguments are place on separate lines. This is more verbose but more easy to recview when changing the signature of functions.

We don't need any rules, just choose what seems most appropriate for the actual context.